opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -214,6 +261,8 @@ namespace Opm {
             unsigned numDof = ebosSimulator_.model().numGridDof();
             wasSwitched_.resize(numDof);
             std::fill(wasSwitched_.begin(), wasSwitched_.end(), false);
+            auto copyHandle = Detail::makeCopyHandle(ebosSimulator_.model().solution(0));
+            grid_.communicate(copyHandle, Dune::InteriorBorder_All_Interface, Dune::ForwardCommunication);
         }
```

Comment:
@blattms: Could you try with ebosSimulator_.model().syncOverlap() instead of 264 and 265? This should do the same an avoid the code above.